lib/autoversion/gitter.rb

```diff
@@ -59,5 +60,14 @@ def write_tag version
       @repo.add_tag "#{@config[:prefix]}#{version}"
     end
 
+    def gitstatus_untracked_workaround
+      gem_rubygit_buggy_untracked_files = @repo.status.untracked.keys
+      git_cmd = "git --work-tree=#{@repo.dir} --git-dir=#{@repo.dir}/.git " +
+                "ls-files -z -d -m -o -X .gitignore"
```

Comment:
It seems this line fails when there's no .gitignore file in the repo. Can it be optional somehow?